package/gluon-mesh-vpn-core/luasrc/lib/gluon/upgrade/500-mesh-vpn

```diff
@@ -5,8 +5,8 @@ local users = require 'gluon.users'
 local util = require 'gluon.util'
 
 local uci = require('simple-uci').cursor()
-local unistd = require 'posix.unistd'
 
+local vpn_core = require 'gluon.mesh-vpn'
 
 uci:section('network', 'interface', 'mesh_vpn', {
 	ifname = 'mesh-vpn',
```

Comment:
It seems like you missed to replace `'mesh-vpn'` here with your newly introduced method `get_interface()`. I already made another commit, where I touched all the instances of this string. See here: https://github.com/freifunk-gluon/gluon/pull/2168/commits/67cd4e0e07b127557b329df56f143462d3c45658 . You can use this commit to cross check.